src/User32.Desktop/User32.cs

```diff
@@ -144,6 +144,17 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern unsafe IntPtr SendMessage(IntPtr hWnd, int wMsg, void* wParam, void* lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern unsafe IntPtr SendMessage(IntPtr hWnd, WindowMessage wMsg, void* wParam, void* lParam);
```

Comment:
That said, per [guidance](https://github.com/AArnott/pinvoke/blob/master/CONTRIBUTING.md#helper-methods) helper methods should be significantly valuable. Simply avoiding a trivial `(int)` cast to the caller is a marginal benefit. We might consider removing the `int` overload where doing so isn't a breaking change.